ui/src/cloud/utils/overrides.ts

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash'
+import HoneyBadger from 'honeybadger-js'
+import {GIT_SHA} from 'src/shared/constants'
+
+if (process.env.CLOUD === 'true') {
```

Comment:
Configure accepts a disabled true/false key where this value could be used.

Reply:
I would prefer we didn't even hit configure when its not running in cloud.